ex11/part1/src/sinkText.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+// import {useState} from 'react';
+
+export default function sinkText(prop) {
```

Comment:
It is good that you created a 2-way communication between a child and parent component, however there is absolutely no need for an extra component here.
It would be best to create simply 5 inputs, even better if you use a loop.